library/common/network/BUILD

```
@@ -76,3 +76,23 @@ envoy_cc_library(
         "@envoy//source/common/network:socket_interface_lib",
     ],
 )
+
+cc_library(
+    name = "ios_platform_verifier",
```

Nit: can we call this `apple_platform_cert_verifier` since "verifier" could imply it verifies a lot of things, and this applies to all Apple platforms, not just iOS, and mirrors the naming of the apple_dns_resolver.

```suggestion
    name = "apple_platform_cert_verifier",
```

---

Good idea. Done.